common/primitives/core/src/identity.rs

```diff
@@ -524,6 +524,18 @@ impl From<[u8; 33]> for Identity {
     }
 }
 
+#[derive(Encode, Decode, TypeInfo, Clone, PartialEq, Eq, RuntimeDebug)]
+pub enum OmniAccountIdentity {
```

Comment:
The name can be a bit misleading - it's meant to be one entry of the IDGraph (I assume).

Maybe `SubIdentity`, or `MemberIdentity`? Hmm - `MemberIdentity` might be better I'm not sure

Reply:
Yes I was not sure about this name either, I had `IDGraphIdentity` before and changed it. I like `MemberIdentity` as well